src/go/rpk/go.mod

```diff
@@ -41,6 +42,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_model v0.6.1
 	github.com/prometheus/common v0.62.0
+	github.com/redpanda-data/common-go/proto v0.0.0-20250416183016-0cd4089182bd
```

Comment:
to pull `*pb.mcp.go` files

this is currently on a branch, i will update before merging